client/components/tinymce/plugins/insert-menu/plugin.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import tinymce from 'tinymce/tinymce';
+import { renderToString } from 'react-dom/server';
+
+import Gridicon from 'components/gridicon';
+
+import menuItems from './menu-items';
+
+const initialize = editor => {
+	menuItems.forEach( item =>
+		editor.addMenuItem( item.name, {
+			classes: 'wpcom-insert-menu__menu-item',
+			cmd: item.cmd,
+			onPostRender() {
+				this.innerHtml( renderToString( item.item ) );
+			}
+		} )
+	);
+
+	editor.addButton( 'wpcom_insert_menu', {
+		type: 'splitbutton',
+		title: 'Insert content',
+		classes: 'btn wpcom-insert-menu insert-menu',
+		cmd: menuItems[0].cmd,
+		menu: menuItems.map( ( { name } ) => editor.menuItems[ name ] ),
+		onPostRender() {
+			const parentNode = this.$el[0].children[0];
```

Comment:
Apparently we can do just this. There was definitely a reason I did it this way, which was because I was losing the drop-down chevron, but changing the code out here and now seems to continue to work okay. I'm not sure if this is a vestige of when I was using a `menubutton` instead of a `splitbutton` or if it's due to the way @drw158 fixed the CSS.

Thanks anyway @aduth - simplified in bee031f